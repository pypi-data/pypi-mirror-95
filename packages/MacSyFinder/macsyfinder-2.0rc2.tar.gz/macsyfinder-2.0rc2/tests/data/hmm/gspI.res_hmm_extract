PSAE001c01_031360	PSAE001c01	3136	gspI	T2SS	1e-43	147.2	0.954128
PSAE001c01_006890	PSAE001c01	689	gspI	T2SS	4.7e-35	119.3	1.000000
PSAE001c01_031350	PSAE001c01	3135	gspI	T2SS	6.6e-08	32.0	0.587156
PSAE001c01_046080	PSAE001c01	4608	gspI	T2SS	4.4e-06	26.1	0.532110
