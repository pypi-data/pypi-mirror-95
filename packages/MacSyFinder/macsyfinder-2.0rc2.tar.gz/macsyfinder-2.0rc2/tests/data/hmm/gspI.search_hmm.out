# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.0 (March 2010); http://hmmer.org/
# Copyright (C) 2010 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  ./profiles/gspI.fasta-aln_edit.hmm
# target sequence database:        datatest/prru_psae.001.c01.fasta
# output directed to file:         ./datatest/res_search/gspI.search_hmm.out
# sequence reporting threshold:    E-value <= 1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       gspI_tcoffee_aln  [M=109]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description
    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------
    8.6e-44  147.4   1.7      1e-43  147.2   1.2    1.1  1  PSAE001c01_031360 C ATG TGA 3479335 3479724 Valid xcpV 390 _
      4e-35  119.6   0.3    4.7e-35  119.3   0.2    1.0  1  PSAE001c01_006890 C ATG TGA 738111 738485 Valid PA0680 375 _
    3.7e-08   32.8   0.4    6.6e-08   32.0   0.2    1.4  1  PSAE001c01_031350 C ATG TGA 3478625 3479338 Valid xcpW 714 _
    2.9e-06   26.7   0.1    4.4e-06   26.1   0.1    1.3  1  PSAE001c01_046080 D ATG TGA 5098709 5099266 Valid pilV 558 _
    1.4e-05   24.5   0.3    2.3e-05   23.8   0.2    1.4  1  PSAE001c01_006860 C ATG TGA 736456 737097 Valid PA0677 642 _
    0.00012   21.5   0.4    0.00016   21.1   0.3    1.3  1  PSAE001c01_027000 C ATG TGA 3020504 3020929 Valid PA2673 426
    0.00031   20.2   0.4    0.00051   19.5   0.2    1.4  1  PSAE001c01_031370 C ATG TGA 3479721 3480239 Valid xcpU 519 _
     0.0024   17.3   0.1      0.005   16.3   0.1    1.6  1  PSAE001c01_046090 D ATG TGA 5099263 5100087 Valid pilW 825 _
     0.0049   16.3   2.6     0.0098   15.3   1.5    1.6  1  PSAE001c01_045800 C ATG TAA 5069082 5069531 Valid pilA 450 _
  ------ inclusion threshold ------
      0.014   14.8   0.1      0.017   14.5   0.1    1.2  1  PSAE001c01_031380 C GTG TGA 3480246 3480692 Valid xcpT 447 _
      0.019   14.4   4.6      0.028   13.9   2.8    1.6  1  PSAE001c01_006870 C GTG TGA 737081 737530 Valid PA0678 450 _
      0.037   13.5   1.0      0.053   13.0   0.7    1.3  1  PSAE001c01_027010 C ATG TGA 3020929 3021339 Valid PA2674 411
      0.066   12.7   0.4      0.093   12.2   0.3    1.4  1  PSAE001c01_046130 D ATG TGA 5104514 5104939 Valid pilE 426 _
       0.25   10.8   3.3        0.3   10.6   2.3    1.3  1  PSAE001c01_006900 D ATG TGA 738719 739168 Valid PA0681 450 _


Domain annotation for each sequence (and alignments):
>> PSAE001c01_031360  C ATG TGA 3479335 3479724 Valid xcpV 390 _PA3099_NP_251789.1_ PA3099 1 3479335 3479724 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  147.2   1.2   1.7e-46     1e-43       1     104 [.       6     110 ..       6     120 .. 0.95

  Alignments for each domain:
  == domain 1    score: 147.2 bits;  conditional E-value: 1.7e-46
   gspI_tcoffee_aln   1 GfTLlEvLvAlaIfalavlavlsavgeslraaerledktlAlwvaenqlaelrlaqaapeagrssgeeevagrewlwrttvsstkdpvlrrv 92 
                        GfTLlEvLvAlaIfa+++++vlsa+++sl++a+rledktlA+w+a+n+l+el+l+q++p++gr++ge e+agr+w+wrt+v+st+++++rrv
  PSAE001c01_031360   6 GFTLLEVLVALAIFAMVAASVLSASARSLQNASRLEDKTLAMWIADNRLNELQLEQTPPSSGRNQGELEFAGRRWEWRTQVDSTAEQDMRRV 97 
                        9******************************************************************************************* PP

   gspI_tcoffee_aln  93 eiaVslk.egrat 104
                         ++V++k  gr++
  PSAE001c01_031360  98 IVWVAAKpLGRER 110
                        *****98666665 PP

>> PSAE001c01_006890  C ATG TGA 738111 738485 Valid PA0680 375 _NP_249371.1_ PA0680 1 738111 738485 | HxcV pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  119.3   0.2   7.9e-38   4.7e-35       1     109 []       9     116 ..       9     116 .. 0.97

  Alignments for each domain:
  == domain 1    score: 119.3 bits;  conditional E-value: 7.9e-38
   gspI_tcoffee_aln   1 GfTLlEvLvAlaIfalavlavlsavgeslraaerledktlAlwvaenqlaelrlaqaapeagrssgeeevagrewlw.rttvsstkdpvlrr 91 
                        GfTL+EvLvAlaI+a+a++a+++avg ++++++ l+dk+lAl++ae++laelrl +  +++g+s++e +++gr +l+ +++v++t+dp+l++
  PSAE001c01_006890   9 GFTLIEVLVALAIVAIALAAAIRAVGLMTDGNGLLRDKSLALLAAESRLAELRLGVG-AAPGNSDFE-CSQGRLRLYcEQRVEATDDPALLW 98 
                        9******************************************************65.9*******7.************************ PP

   gspI_tcoffee_aln  92 veiaVslkegratslasl 109
                        vei+V++++++a++la+l
  PSAE001c01_006890  99 VEIRVRTQREQAVPLARL 116
                        ***********9998875 PP

>> PSAE001c01_031350  C ATG TGA 3478625 3479338 Valid xcpW 714 _PA3098_NP_251788.1_ PA3098 1 3478625 3479338 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   32.0   0.2   1.1e-10   6.6e-08       1      64 [.       6      71 ..       6      91 .. 0.71

  Alignments for each domain:
  == domain 1    score: 32.0 bits;  conditional E-value: 1.1e-10
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavge..slraaerledktlAlwvaenqlaelrlaqaapeagrs 64
                       GfTLlE+L+A+aIfal +la+ ++  +  ++++a r++++ + ++v    + e +l qa+++++r 
  PSAE001c01_031350  6 GFTLLELLIAIAIFALLALATYRMFDSvmQTDQATRVQEQRMRELVRAMGALERDLTQAVERPVRD 71
                       9******************99887665225566677777777666666666666666666665554 PP

>> PSAE001c01_046080  D ATG TGA 5098709 5099266 Valid pilV 558 _PA4551_NP_253241.1_ PA4551 1 5098709 5099266 | type 4 fi
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   26.1   0.1   7.5e-09   4.4e-06       1      58 [.      14      71 ..      14      95 .. 0.89

  Alignments for each domain:
  == domain 1    score: 26.1 bits;  conditional E-value: 7.5e-09
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavgeslraaerledktlAlwvaenqlaelrlaqaa 58
                       Gf+++EvLvAl  +++ vl+++ + g+  +   +  ++  A++++ n l+++r + +a
  PSAE001c01_046080 14 GFSMIEVLVALLLISIGVLGMIAMQGKTIQYTADSVERNKAAMLGSNLLESMRASPKA 71
                       9************************************************999988765 PP

>> PSAE001c01_006860  C ATG TGA 736456 737097 Valid PA0677 642 _NP_249368.1_ PA0677 1 736456 737097 | HxcW pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   23.8   0.2   3.9e-08   2.3e-05       1      52 [.      16      70 ..      16      94 .. 0.71

  Alignments for each domain:
  == domain 1    score: 23.8 bits;  conditional E-value: 3.9e-08
   gspI_tcoffee_aln  1 GfTLlEvLvA...laIfalavlavlsavgeslraaerledktlAlwvaenqlael 52
                       GfTL+Ev+vA   +aI++l +   l +++++ +  e+  ++  Al+ a nql+  
  PSAE001c01_006860 16 GFTLIEVMVAimlMAIVSLMAWRGLDSIARASAHLEDSTEQGAALLRALNQLERD 70
                       9*********555566666677777777777777777778889999999999753 PP

>> PSAE001c01_027000  C ATG TGA 3020504 3020929 Valid PA2673 426 _NP_251363.1_ PA2673 1 3020504 3020929 | type II secret
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   21.1   0.3   2.7e-07   0.00016       1      81 [.       7      84 ..       7      95 .. 0.64

  Alignments for each domain:
  == domain 1    score: 21.1 bits;  conditional E-value: 2.7e-07
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavgeslraaerledktlAlwvaenqlaelrlaqaapeagrssgeeevagrewlwrttv 81
                       GfTLlE  vAl+ +a++  a+    +++ r  +r+e+  l    a+ ++a l+ +   p+  + +g  +    + +wr+ +
  PSAE001c01_027000  7 GFTLLEAVVALTLLAVVGGALFAWLNSAFRSMQRVEAAELRIETARVAMAYLERM--NPA-LEPAGRVRLGHYRLEWRSSP 84
                       9*********999999888888887777777777777776655554444443333..232.23334445555556665555 PP

>> PSAE001c01_031370  C ATG TGA 3479721 3480239 Valid xcpU 519 _PA3100_NP_251790.1_ PA3100 1 3479721 3480239 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   19.5   0.2   8.5e-07   0.00051       1      45 [.       6      51 ..       6      75 .. 0.90

  Alignments for each domain:
  == domain 1    score: 19.5 bits;  conditional E-value: 8.5e-07
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalav.lavlsavgeslraaerledktlAlwva 45
                       GfTL+E++v + I+++ + lavls    s++ + + e++ lA++++
  PSAE001c01_031370  6 GFTLIELMVVMVIISVLIgLAVLSTGFASTSRELDSEAERLAGLIG 51
                       9**************977589999999999999999******9985 PP

>> PSAE001c01_046090  D ATG TGA 5099263 5100087 Valid pilW 825 _PA4552_NP_253242.1_ PA4552 1 5099263 5100087 | type 4 fi
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   16.3   0.1   8.5e-06     0.005       1      66 [.      12      84 ..      12     105 .. 0.75

  Alignments for each domain:
  == domain 1    score: 16.3 bits;  conditional E-value: 8.5e-06
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavgeslra.......aerledktlAlwvaenqlaelrlaqaapeagrssg 66
                       G++++E+LvAlaI +  +l++ ++   + r+       a+  e+   A++  ++qla+   + +a ++ + ++
  PSAE001c01_046090 12 GLSMIELLVALAISSFLILGITQIYLDNKRNylfqqgqAGNQENGRFAMMFLDQQLAKVGFRRRADDPNEFAF 84
                       89****************9999987766654333333367788889999999999999888877665555444 PP

>> PSAE001c01_045800  C ATG TAA 5069082 5069531 Valid pilA 450 _PA4525_NP_253215.1_ PA4525 1 5069082 5069531 | type 4 fi
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   15.3   1.5   1.6e-05    0.0098       1      26 [.       6      31 ..       6      47 .. 0.77

  Alignments for each domain:
  == domain 1    score: 15.3 bits;  conditional E-value: 1.6e-05
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavg 26
                       GfTL+E+++ +aI+++ ++ ++   +
  PSAE001c01_045800  6 GFTLIELMIVVAIIGILAAIAIPQYQ 31
                       9*************987766655444 PP

>> PSAE001c01_031380  C GTG TGA 3480246 3480692 Valid xcpT 447 _PA3101_NP_251791.1_ PA3101 1 3480246 3480692 | general s
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   14.5   0.1   2.9e-05     0.017       1      31 [.      14      45 ..      14      87 .. 0.75

  Alignments for each domain:
  == domain 1    score: 14.5 bits;  conditional E-value: 2.9e-05
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavl.savgeslra 31
                       GfTL+E++v + I+++ ++ v+ +++++  +a
  PSAE001c01_031380 14 GFTLIEIMVVVVILGILAALVVpQVMSRPDQA 45
                       9************9976555441555554444 PP

>> PSAE001c01_006870  C GTG TGA 737081 737530 Valid PA0678 450 _NP_249369.1_ PA0678 1 737081 737530 | HxcU pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   13.9   2.8   4.7e-05     0.028       1      23 [.       8      29 ..       8      52 .. 0.82

  Alignments for each domain:
  == domain 1    score: 13.9 bits;  conditional E-value: 4.7e-05
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavls 23
                       GfTL+E++v l I+++a +a +s
  PSAE001c01_006870  8 GFTLIELMVVLVIVGIATAA-IS 29
                       9**************98766.44 PP

>> PSAE001c01_027010  C ATG TGA 3020929 3021339 Valid PA2674 411 _NP_251364.1_ PA2674 1 3020929 3021339 | type II secret
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   13.0   0.7   8.9e-05     0.053       2      25 ..      15      39 ..      14      54 .. 0.70

  Alignments for each domain:
  == domain 1    score: 13.0 bits;  conditional E-value: 8.9e-05
   gspI_tcoffee_aln  2 fTLlEvLvAlaIfal.avlavlsav 25
                       fTLlE+Lv l I+++ a++a+  +v
  PSAE001c01_027010 15 FTLLELLVVLVIVGAiAAVALPGLV 39
                       9***********9651555544444 PP

>> PSAE001c01_046130  D ATG TGA 5104514 5104939 Valid pilE 426 _PA4556_NP_253246.1_ PA4556 1 5104514 5104939 | type 4 fi
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   12.2   0.3   0.00016     0.093       1      38 [.       7      44 ..       7     101 .. 0.74

  Alignments for each domain:
  == domain 1    score: 12.2 bits;  conditional E-value: 0.00016
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavlsavgeslraaerledk 38
                       GfTLlE+ v +a++++ +  ++ + ++    ++r e +
  PSAE001c01_046130  7 GFTLLEMVVVVAVIGILLGIAIPSYQNYVIRSNRTEGQ 44
                       9**************99887777777666555555544 PP

>> PSAE001c01_006900  D ATG TGA 738719 739168 Valid PA0681 450 _NP_249372.1_ PA0681 1 738719 739168 | HxcT pseudopilin  
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   10.6   2.3    0.0005       0.3       1      22 [.      18      39 ..      18     101 .. 0.82

  Alignments for each domain:
  == domain 1    score: 10.6 bits;  conditional E-value: 0.0005
   gspI_tcoffee_aln  1 GfTLlEvLvAlaIfalavlavl 22
                       GfTL+E++v + I+++ ++ v+
  PSAE001c01_006900 18 GFTLIEIMVVVVILGILAAMVV 39
                       9**********99987766554 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (109 nodes)
Target sequences:                       8332  (2926593 residues)
Passed MSV filter:                       405  (0.0486078); expected 166.6 (0.02)
Passed bias filter:                      291  (0.0349256); expected 166.6 (0.02)
Passed Vit filter:                        38  (0.00456073); expected 8.3 (0.001)
Passed Fwd filter:                        14  (0.00168027); expected 0.1 (1e-05)
Initial search space (Z):               8332  [actual number of targets]
Domain search space  (domZ):              14  [number of targets reported over threshold]
# CPU time: 0.12u 0.01s 00:00:00.13 Elapsed: 00:00:00.05
# Mc/sec: 6379.97
//
